mlir-tensorrt/compiler/lib/Compiler/CMakeLists.txt

```diff
@@ -19,6 +20,8 @@ add_mlir_tensorrt_library(MLIRTensorRTCompilerStableHloToExecutable
     StableHloToExecutable.cpp
     # TODO: TensorRTExtension should be an independent library.
     TensorRTExtension/TensorRTExtension.cpp
+    # TODO (pranavm): TensorRTToExecutable should probably be a separate library
+    TensorRTToExecutable.cpp
```

Comment:
Given that there's an open TODO for `TensorRTExtension.cpp` I'm wondering if this is more complicated than it seems.